src/libs/ajax/submissions/Submissions.ts

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash/fp';
+import { authOpts } from 'src/auth/auth-session';
+import { fetchRawls } from 'src/libs/ajax/ajax-common';
+
+export const Submissions = (signal?: AbortSignal) => ({
```

Comment:
I found no current uses of this.... Should I completely remove Submissions module?

Reply:
ok, will remove it